### PR TITLE
KTLN-154 : Collection Aggregate Operations in Kotlin

### PR DESCRIPTION
**Pull request by anirban99**

### CODE REVIEW

**Comment by anirban99** [comment]
I removed the generic prints. But I would prefer to keep these `println("total = $total, it = $it")`. The reason is that when we run the program/tests, it helps to understand the sequence of elements being called for the different fold and reduce functions. What do you think about it?

File: core-kotlin-modules/core-kotlin-collections/src/main/kotlin/com/baeldung/kotlin/collections/AggregateOperations.kt [hunk 1]
```diff
@@ -0,0 +1,107 @@
+package com.baeldung.kotlin.collections
```
Comment (KevinGilmore) [inline core-kotlin-modules/core-kotlin-collections/src/main/kotlin/com/baeldung/kotlin/collections/AggregateOperations.kt:1]: Let's make the package more specific, e.g. com.baeldung.kotlin.collections.aggregate

Comment (anirban99) [reply core-kotlin-modules/core-kotlin-collections/src/main/kotlin/com/baeldung/kotlin/collections/AggregateOperations.kt]: @KevinGilmore Moved it to a new module.

**Comment by rpvilao** [comment]
@anirban99 there was an issue with the refactor, it literally created a folder with dots instead of the structure

**Comment by anirban99** [comment]
@rpvilao Where did it create the folder with dots? Can you please point it out in the code?

**Comment by rpvilao** [comment]
image
Aggregate Operations for example.

**Comment by anirban99** [comment]
@rpvilao Fixed the package structure

**Pull request merged**